beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -582,6 +582,20 @@ pub static FORK_CHOICE_PROCESS_ATTESTATION_TIMES: LazyLock<Result<Histogram>> =
             "Time taken to add an attestation to fork choice",
         )
     });
+pub static FORK_CHOICE_READ_LOCK_AQUIRE_TIMES: LazyLock<Result<Histogram>> = LazyLock::new(|| {
+    try_create_histogram_with_buckets(
+        "beacon_fork_choice_read_lock_aquire_seconds",
+        "Time taken to aquire the fork-choice read lock",
+        exponential_buckets(1e-3, 2.0, 10),
```

Comment:
I think we could start the lower end at 0.1ms, as I think on healthy nodes the lock should take a lot less than 1ms to acquire? We could also set the exponential factor to 4x so that it ramps up quicker?

```suggestion
        exponential_buckets(1e-4, 4.0, 7),
```